styles/_navbar.less

```diff
@@ -116,6 +117,37 @@
 body.site-menu-open .site-menu-toggle {
   .hamburger-button-open();
 }
+.navbar-top-section{
```

Comment:
I don't have specific code suggestions, but if we decide to stick with a second row of nav, there is still CSS work to be done here. Right now, the additional bar pushes the original bar down further, masking the top of the content. 

Here's what it looked like before these changes: 

<img width="1604" alt="image" src="https://github.com/camunda/camunda-docs-theme/assets/1627089/be61a760-71a8-409a-bebe-b406c93b795d">

And here's what these changes introduce: 

<img width="1521" alt="image" src="https://github.com/camunda/camunda-docs-theme/assets/1627089/d976e880-5b5c-4ac5-adac-c8612ba71019">

I'm unable to scroll far enough on the page to see those top bits.

Reply:
This is fixed!

Reply:
👍 

For others playing along, this is what it looks like now: 

<img width="1596" alt="image" src="https://github.com/camunda/camunda-docs-theme/assets/1627089/dc058434-b2d7-4585-bfdc-dff97fe59419">